### DISCUSSION TITLE
EODAG vs. datalad

### DISCUSSION THREAD

**Discussion by mankoff** [CS-SI/eodag#409] [Q&A]
Hello, I just discovered STAC, and the EODAG tool as an implementation of the STAC spec. I was working on a data meta-portal (not a meta-data portal) using datalad https://www.datalad.org/ (built on git annex). datalad and EODAG seem similar, but I'm new enough to not fully understand either. Can anyone here provide some insight into pros/cons of datalad vs. EODAG, complementary or divergent use cases, etc.? Thanks.

**Comment by sbrunato** [comment]
Hello @mankoff , I don't know much about datalad, but it seems that it is more focused on abstracting, cloning, sharing datasets where eodag is a sort of proxy to data providers APIs, giving users a unique API to search and download EO products.
See https://eodag.readthedocs.io/en/stable/getting_started_guide/overview.html for an overview of `eodag`, and you can browse the online documentation where you will find lots of useful information and examples
